case-server/src/main/java/com/powsybl/caseserver/CaseInfos.java

```diff
@@ -18,23 +20,40 @@
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-@AllArgsConstructor
-@NoArgsConstructor
-@Getter
+@SuperBuilder
+@Getter()
 @ApiModel("Case infos")
 public class CaseInfos {
 
-    private static final String NAME_HEADER_KEY     = "name";
-    private static final String FORMAT_HEADER_KEY   = "format";
+    protected static final String NAME_HEADER_KEY   = "name";
+    protected static final String UUID_HEADER_KEY   = "uuid";
+    protected static final String FORMAT_HEADER_KEY = "format";
 
-    private String name;
-    private String format;
-    private UUID uuid;
+    @NonNull protected String   name;
+    @NonNull protected UUID     uuid;
+    @NonNull protected String   format;
 
-    public static Message<String> getMessage(CaseInfos caseInfos) {
+    public static CaseInfos create(String fileBaseName, UUID caseUuid, String format, FileNameInfos fileNameInfos) {
+        switch (fileNameInfos.getType()) {
+            case ENTSOE:
+                EntsoeFileName entsoeFileName = (EntsoeFileName) fileNameInfos;
```

Comment:
I am not sure this is the right place for entsoe related code. What about if we would like to move FileName impl in another Maven module to really leverage plugins archi?

Reply:
It is to fix the error when we get a case with xiidm format :
curl -X GET "http://localhost:5000/v1/cases/c23b830e-92aa-4804-b911-614dc541349b?xiidm=true" -H "accept: */*"
com.powsybl.commons.PowsyblException: No implementation available for triple store rdf4j

Reply:
A 'entsoe' subpackage has been created int the package 'parser' and the package 'caseserver'